pywbem/_subscription_manager.py

```diff
@@ -1259,8 +1316,31 @@ def _create_destination(self, server_id, dest_url, owned,
             URL specify the target location to be used by the WBEM server.
 
           owned (:class:`py:bool`):
-            Defines whether or not the created instance is *owned* by the
-            subscription manager.
+            Defines the ownership type of the created listener destination
```

Comment:
Shouldn't this be ownership rather than ownership type.

Reply:
see response to previous occurrence of this comment.